src/main/java/games/strategy/engine/lobby/server/db/UserController.java

```diff
@@ -126,6 +149,29 @@ public void createUser(final DBUser user, final HashedPassword hashedPassword) {
   }
 
   @Override
+  public void createUser(final DBUser user, final String password) {
+    Preconditions.checkState(user.isValid(), user.getValidationErrorMessage());
+
+    try (final Connection con = connectionSupplier.get()) {
+      try (final PreparedStatement ps = con.prepareStatement(
+          "insert into ta_users (username, password, email, joined, lastLogin, admin) values (?, ?, ?, ?, ?, ?)")) {
+        ps.setString(1, user.getName());
+        ps.setString(2, BCrypt.hashpw(password, BCrypt.gensalt()));
```

Comment:
The salt is stored with the string, similar to MD5Crypt.
The structure of the String returned by `BCrypt#hashpw()` is `$2a$<salt>$<hash>`